### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by shiryarm** [runeharlyk/SpotMicroESP32-Leika#89] [Q&A] [answered]
I am looking at the circuit diagram and I noticed that some of the wires aren't connected to anything at all. Why is that? And how does it work?

**Comment by runeharlyk** [comment, marked as answer]
Hi.
So the wires that ends in a header eg: 
![image](https://github.com/user-attachments/assets/9a2ec260-682e-47ad-a6eb-229b44186e53)

should be connected to the other wires with he same color. It's just to simplify the schematic, as the VCC, GND, SDA and SCL wires are use to connect most components together.

This mean:
- All the `white` wires should be connected together
- All the `grey` wires should be connected together
- All the `pink` wires should be connected together
- All the `black` wires should be connected together
- All the `purple` wires should be connected together
